uobtheatre/payments/transferables.py

```diff
@@ -0,0 +1,91 @@
+import math
+from abc import abstractmethod
+from typing import TYPE_CHECKING, Optional
+
+from django.contrib.contenttypes.models import ContentType
+from django.db import models
+
+from uobtheatre.payments.models import SalesBreakdown, Transaction
+from uobtheatre.payments.payables import Payable
+
+if TYPE_CHECKING:
+    pass
+
+# pylint: disable=abstract-method
+class Transferable(Payable):
+    """
+    A transferable is a payable which can be transfered. A transfer is a
+    to/from another transferable of the same type. For bookings this enables bookings to be updated (chnaged perfomrance)
+    """
+
+    # Whether the booking has be transferred to
+    transfered_from = models.OneToOneField(
+        "self", on_delete=models.RESTRICT, null=True, related_name="_transfered_to"
+    )
+
+    @property
+    def transfered_to(self) -> Optional["Transferable"]:
```

Comment:
```suggestion
    def transferred_to(self) -> Optional["Transferable"]:
```

Reply:
There are a lot of other instances where you have spelt it `transfered`. It should be `transferred`